composer.json

```diff
@@ -9,7 +9,7 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": ">=7.4, <8.2"
+        "php": "~8.1.0"
```

Comment:
The amount of time to keep on par with PHP 8.0 whilst also having ENUM abstractions is just not worth it.